distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -863,6 +869,9 @@ public void testUpdateContentResourceUri() throws IOException {
 
     @Test
     public void testCachedContentLengthHeader() throws IOException {
+
+        console.runCommand(CLEAR_CACHE);
```

Comment:
Karaf commands should be executed synchronously, however I know solr sometimes has a very brief delay on operations not involving a specific metacard id. Clearing the cache here may even be somewhat excessive considering we are adding the content and then operating with its metacard id specifically. 

Since we don't have a definitive reason for the test instability, I added this more as a precaution.